controllers/pokemon_controller.go

```diff
@@ -39,3 +53,13 @@ func FindPokemonById(c *gin.Context) {
 	}
 	c.JSON(http.StatusNotFound, gin.H{"message": "pokemon not found"})
 }
+
+func (pks PokemonServiceHandler) LoadPokemons(c *gin.Context) {
+	_, error := pks.pokemonService.LoadPokemons()
```

Comment:
Usually errors are called err, even though error is not necessarilly a reserved word, it's used only to specify the error data type